site/content/en/docs/tasks/run_rayjobs.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Run A RayJob"
+date: 2023-05-18
+weight: 6
+description: >
+  Run a Kueue scheduled RayJob.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [KubeRay's](https://ray-project.github.io/kuberay/) [RayJob](https://ray-project.github.io/kuberay/guidance/rayjob/).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+By default the integration for `ray.io/rayjob` is not enabled, check the [Installation](/docs/installation/) for details on [Kueue's manger configuration](/docs/installation/#install-a-custom-configured-released-version).
+
+Check [administer cluster quotas](/docs/tasks/administer_cluster_quotas) for details on the initial cluster setup.
+
+Check [KubeRay](https://ray-project.github.io/kuberay/) documentation for installation and configuration details.
+
+## RayJob definition
+
+Besides the usual RayJob configuration while using Kueue take into consideration the following aspects:
+
+### a. Queue selection
+
+The target [local queue](/docs/concepts/local_queue) should be specified in the `metadata.labels` section of the RayJob configuration.
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.rayClusterSpec`.
+
+```yaml
+    headGroupSpec:
+      template:
+        spec:
+          containers:
+            - resources:
+                requests:
+                  cpu: "1"
+    workerGroupSpecs:
+      - template:
+          spec:
+            containers:
+              - resources:
+                  requests:
+                    cpu: "1"
+```
+
+### c. Limitations
+
+- A Kueue managed RayJob cannot use an existing cluster.
```

Comment:
```suggestion
- A Kueue managed RayJob cannot use an existing RayCluster.
```

Maybe also add a link to what a Ray cluster is. Otherwise it can be confused with kubernetes cluster.